create_profile_without_manual_controls/parameter.json

```diff
@@ -0,0 +1,7 @@
+{
+  "IBMCLOUD_API_TOKEN": "",
+  "SCC_INSTANCE_ID": "",
+  "BASE_PROFILE_ID": "",
+  "COMMAND_CENTER_URL": "https://us-south.compliance.test.cloud.ibm.com"
```

Comment:
Please remove test.cloud as it's internal and can't be exposed

Reply:
Closing this PR, migrated changes at https://github.com/IBM/security-compliance-automation/pull/6